### PR TITLE
Updated int.m3u

### PR DESCRIPTION
**Pull request by themak1985**
Removed Broken Links, Added Missing Languages, fixed incorrect languages.
A lot of those vietnamese channels are online only channels - my guess is that they should be moved to vn.m3u but I wanted to do this first.

### CODE REVIEW

File: channels/int.m3u [hunk 1]
```diff
@@ -27,33 +27,19 @@ https://www.ast.tv/stream/2/ultra.m3u8
...
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="XXX",Big Tits HD
-http://nruxmzi.ojswi5dsmftgm2ldfz4hs6q.cmle.ru/bigtits.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="XXX",BIGASS
-http://nruxmzi.ojswi5dsmftgm2ldfz4hs6q.cmle.ru/bigass.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="XXX",BIGDICK
-http://nruxmzi.ojswi5dsmftgm2ldfz4hs6q.cmle.ru/bigdick.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="XXX",BLONDE
-http://nruxmzi.ojswi5dsmftgm2ldfz4hs6q.cmle.ru/blonde.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="XXX",BLOWJOB
-http://nruxmzi.ojswi5dsmftgm2ldfz4hs6q.cmle.ru/blowjob.m3u8
```
Comment (freearhey) [inline channels/int.m3u]: Could you explain why you deleted those links?

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by freearhey** [comment]
As for the Vietnamese channels, I agree, it would be better to move them to the appropriate playlist.

File: channels/int.m3u (hunk 1 above)
Comment (themak1985) [reply channels/int.m3u]: Did they work for you? I couldn't get them to work - even when I used streamtest.in

Comment (themak1985) [reply channels/int.m3u]: Looks like they are geoblocked. I just changed my vpn to Ukraine and it worked - US, UK, Slovakia, South Africa, all failed. I don't have a russian VPN to check if that could work but based on the .ru address - I will assume yes.

Comment (freearhey) [reply channels/int.m3u]: You're right, links only work from some locations.

**Pull request merged**